addons/docs/src/frameworks/web-components/config.js

```diff
@@ -24,7 +24,7 @@ addParameters({
     page: DocsPage,
     extractProps: tagName => {
       const customElements = getCustomElements();
-      if (customElements) {
+      if (typeof tagName === 'string' && customElements && customElements.tags) {
```

Comment:
Maybe `checkValidComponent(tagName)` that throws if it's not a string?

Reply:
if nothing is provided we should not throw...

but if something is provided which is different then a string then we should 👍 
added 💪